src/poolboy.erl

```diff
@@ -383,13 +379,7 @@ handle_worker_exit(Pid, State) ->
            monitors = Monitors,
            idle_workers = IdleWorkers,
            overflow = Overflow} = State,
-    NewIdleWorkers = case maps:get(Pid, IdleWorkers, undefined) of
-            undefined ->
-                IdleWorkers;
-            ref ->
-                erlang:cancel_timer(ref),
-                maps:remove(Pid, IdleWorkers)
```

Comment:
Just noticed this clause would never match: it's matching `ref` atom not on a reference.

Reply:
🤦